pennylane_forest/qpu.py

```diff
@@ -19,29 +19,24 @@
 Code details
 ~~~~~~~~~~~~
 """
-import re
 import warnings
-import pkg_resources
-from packaging import version
 
 import numpy as np
 from pyquil import get_qc
+from pyquil.experiment import SymmetrizationLevel
 from pyquil.operator_estimation import (
     Experiment,
     ExperimentSetting,
     TensorProductState,
     group_experiments,
     measure_observables,
 )
-from pyquil.paulis import sI, sX, sY, sZ
+from pyquil.paulis import sI, sZ
 from pyquil.quil import Program
 from pyquil.quilbase import Gate
 
 from pennylane.operation import Tensor
 
-from typing import List
-
-from ._version import __version__
 from .qvm import QVMDevice
```

Comment:
It is weird that the `QPUDevice` inherits from the `QVMDevice`... In the future we could either move the common methods to the `ForestDevice` class or create a new class

Reply:
Agreed, that caught me off guard as well. I can create a followup issue if you'd like.